apps/web/src/components/ExperienceFormFields/WorkFields/GovFields.tsx

```diff
@@ -216,8 +205,13 @@ const GovFields = ({ labels }: SubExperienceFormProps) => {
       resetDirtyField("classificationLevel");
     }
 
-    if (watchGovEmploymentType) {
-      resetDirtyField("govPositionType");
+    /**
+     * Wipe position type field if employment type changes, setting it to null rather than just reset incase the user default is bad
+     */
+    if (
+      watchGovEmploymentType !== WorkExperienceGovEmployeeType.Indeterminate
+    ) {
+      resetField("govPositionType", { keepDirty: false, defaultValue: null });
```

Comment:
It is possible a user has invalid state experiences, term with a position type value. Resetting the field just resets it to whatever it initialized with, which breaks the form. Need to specifically set it to null to handle such cases. 

I don't think those invalid state experiences present much of a problem, if any exist, just something to be aware of